cmd/template/cluster/flag.go

```diff
@@ -112,6 +129,12 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.OpenStack.RootVolumeSourceUUID, flagOpenStackRootVolumeSourceUUID, "", "Root volume source UUID (OpenStack only).")
 	cmd.Flags().StringVar(&f.OpenStack.NodeCIDR, flagOpenStackNodeCIDR, "", "CIDR used for the nodes.")
 
+	// OpenStack App only.
+	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.1.0", "Cluster App version to be installed. (OpenStack App CR only).")
```

Comment:
I think we want to hide these flags for now, see https://github.com/giantswarm/kubectl-gs/pull/540#issuecomment-977888540 and https://github.com/giantswarm/roadmap/issues/605. WDYT @giantswarm/team-rocket?

Reply:
I would suggest this, too.

Reply:
Ok, it's done